internal/configuration/configuration_test.go

```diff
@@ -17,7 +17,7 @@ type config struct {
 	JC_OPENSHIFT_API_TOKEN   string
 	JC_AUTH_TOKEN            string
 	JC_IDLE_AFTER            string
-	JC_UN_IDLE_RETRY         string
+	JC_MAX_RETRIES           string
```

Comment:
MIssing JC_CHECK_INTERVAL here?

Reply:
Technically not required, but I can add it